apps/cnquery/cmd/plugin.go

```diff
@@ -109,6 +121,20 @@ func (c *cnqueryPlugin) RunQuery(conf *proto.RunQueryConfig, out shared.OutputHe
 		out.WriteString("[")
 	}
 
+	var upstreamConfig *resources.UpstreamConfig
+	serviceAccount := opts.GetServiceCredential()
+	if serviceAccount != nil {
+		certAuth, _ := upstream.NewServiceAccountRangerPlugin(serviceAccount)
```

Comment:
Is it really not possible for this to error because of the ReadConfig() above, or should this potential error case be handled?

Reply:
Nice catch. Thanks.

That was a problem in multiple commands. I added a check.